### ISSUE TITLE
Optional enum types require both "x-nullable" and "null" to pass validation

### ISSUE THREAD

**Issue by dhojnowski** [Yelp/bravado-core#151]
I have a swagger spec that specifies an enum that I'd like to be nullable.  I would expect that I could just add "x-nullable" to this, but it appears I have to also add null to the list of values for the enum

```
"DeviceType": {
      "type": "string",
      "x-nullable": true,
      "enum": [
        "android",
        "iphone",
        "ipad"
      ]
    },
```
gives `None is not one of [u'android', u'iphone', u'ipad']`

```
"DeviceType": {
      "type": "string",
      "enum": [
        "android",
        "iphone",
        "ipad",
        null
      ]
    },
```
gives `None is not of type u'string'`  Only 
```
"DeviceType": {
      "type": "string",
      "x-nullable": true,
      "enum": [
        "android",
        "iphone",
        "ipad",
        null
      ]
    },
``` 
passes.

This is on bravado-core 4.5.1

**Comment by macisamuele** [comment]
Hey @dhojnowski the issue is fixed by #145 .
The new version will be released soon.

**Comment by sjaensch** [comment]
Actually @dhojnowski that fix is in the 4.6.1 release, unfortunately it didn't make it into the release notes. Please give it a try and reopen this issue if it's not fixed.

**Comment by dhojnowski** [comment]
Yep, fixed.  Thanks for getting back so quickly on this!
